ref_builder/otu/modify.py

```diff
@@ -150,13 +149,13 @@ def resize_monopartite_plan(
     accessions: list[str],
```

Comment:
I'm not sure. I feel like the change should remain pretty deliberate, but perhaps reckless plan expansion is not a realistic thing to worry about?

Reply:
Found a potential problem with just reusing `add_segments_to_plan()`.

if the original segment had a null `.name` (which is very likely), the new extended `Plan` can't pass validation.

Reply:
Then the user has to set a name for the existing segment before adding another. Right?
Isn't that the same flow that would have happened before?

Reply:
Apparently not. `otu.modify.add_segments_to_plan()` doesn't do anything to currently existing segments, so the `ref-builder otu plan extend` command doesn't include a `--name` option.

I think `resize_monopartite_plan()` and the `ref-builder otu plan multipartite` command are going to need revamping for sure, but it might be best to leave that kind of work to a second PR. Alas the function is not completely redundant yet.

(Incidentally I've been trying to think of an elegant single-line solution for adding multiple segments with custom names for a while, but it's eluded me thus far.)

Reply:
OK. I will bear that in mind while reviewing.